invoices/invoiceregistry.go

```diff
@@ -196,6 +259,29 @@ func (i *InvoiceRegistry) invoiceEventNotifier() {
 				i.singleNotificationClients[e.id] = e
 			}
 
+		// A new htlc came in for auto-release.
+		case event := <-i.htlcAutoReleaseChan:
+			log.Debugf("Scheduling auto-release for htlc: "+
```

Comment:
Shouldn't we use either signed or unsigned types for block height everywhere?

Reply:
Yes, that is inconsistent at the moment. But if you start changing, you will expand into many different areas and also replace one case by the other sometimes.